proposals/new-ipfs-gc.md

```diff
@@ -0,0 +1,103 @@
+# Project Pitch: IPFS Garbage Collection v2
+###### Making GC fast, efficient, and targeted
+
+Authors: @gammazero
+
+## Purpose and Impact
+
+### Background and Intent
```

Comment:
I think more anecdotes/justification from the conversation in https://protocollabs.slack.com/archives/G01KZD3FETY/p1613779249428300 can be added.  The transitive impact that this has on NFTs is important as we don't want to hamper that usecase.

Reply:
This link did not work for me, but I added it to related discussions below

Reply:
This link did not work for me, but I added it to the "Related discussions" section anyway